launch_mut_type.h

```diff
@@ -0,0 +1,107 @@
+#ifndef LAUNCH_MUT_TYPE_H
```

Comment:
I am not sure why you put all these functions in seperate header files, is there a good reason for this?

Reply:
I mean why you didn't put launch_env_change and launch_mut_type in the same file

Reply:
Also the file for selection type is not named consistently with the others, which is confusing

Reply:
The selction file is not named consistently because I first just tried to move them all in the same file, and renaming files is suhc a bother that I left it as it was.